content/cybersecurity/concepts/nmap/terms/udp-scan/udp-scan.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: 'UDP Scan' 
+Description: 'Scans (only) the UDP Ports of a target, which aren't scanned in a regular scan. Can be combined with any TCP scan' 
```

Comment:
```suggestion
Description: 'Scans (only) the UDP Ports of a target, which are not scanned in a regular scan. Can be combined with any TCP scan' 
```

Reply:
The Metadata doesn't like the apostrophe in the description.

Reply:
Thank you very much! That would have taken forever for me to find out